hpms/pom.xml

```diff
@@ -50,11 +50,7 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-webflux</artifactId>
         </dependency>
-        <dependency>
```

Comment:
This dependency isn't used and pulls in dependencies that are insecure. 
https://mvnrepository.com/artifact/org.projectreactor/reactor-spring/1.0.1.RELEASE